convert/convert_windows.ml

```diff
@@ -396,9 +396,59 @@ let convert (g : G.guestfs) source inspect i_firmware
   and configure_pnputil_install () =
     let fb_script = "@echo off\n\
                      \n\
-                     echo Wait for VirtIO drivers to be installed\n\
-                     %systemroot%\\Sysnative\\PnPutil -i -a \
-                     %systemroot%\\Drivers\\Virtio\\*.inf" in
+                     setlocal EnableDelayedExpansion\n\
+                     set log=%~dp0pnputil.log\n\
```

Comment:
You don't need to create another log file.  The output of the script is already logged, so just print stuff normally and it'll appear in the central log.